project-codeflare/multi-cluster-app-dispatcher
# Enhance MCAD Logs and fix naming consistency

# Issue link
Resolves #644

### Motivation
In the scenario where we want to create an AW in two different namespaces but with the same name, it is important to know which AppWrapper the logs are referring to.

Previously: 
- There are a number of instances where the logs only provide the `name` of the AppWrapper or Job, instead of following the most used pattern of `namespace/name` as done i.e., [here](https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/b445015360abf09bf11bb1c7024068005a1a8a87/pkg/controller/queuejob/queuejob_controller_ex.go#L595). 

- There are a couple of places where the logs display `name/name`, such as [here](https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/b445015360abf09bf11bb1c7024068005a1a8a87/pkg/controller/queuejob/queuejob_controller_ex.go#L1407).

- A few places have the `name/namespace` pattern, which can be inconsistent when looking/searching through the logs.

- Few instances where i.e., `qj.Name` is provided in [the Info log](https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/b445015360abf09bf11bb1c7024068005a1a8a87/pkg/controller/queuejob/queuejob_controller_ex.go#L1662) but not used.

# What changes have been made

- The logs now consistently refer to the AppWrappers or Jobs in the form of `namespace/name`.
- Typo fixes.



# Verification steps



- With MCAD running, create an AppWrapper, and check MCAD logs for `<namespace/name>` of your AW.

## Checks
- Testing Strategy
